README.md

```diff
@@ -11,7 +11,7 @@ This repository contains the code of the backend used by the
 
 ## Table of content
 
-- [IO mobile backend](#io-mobile-backend)
+- [Digital citizenship web and mobile backend](#digital-citizenship-web-and-mobile-backend)
```

Comment:
why this regression? (maybe need rebasing)

Reply:
Rebased :)

Reply:
I this to be resolved yet?